portfolio/src/main/webapp/index.html

```diff
@@ -2,16 +2,24 @@
 <html>
   <head>
     <meta charset="UTF-8">
+    <img src="/Images/Name.png">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1 style="background-color:Violet;">
```

Comment:
I don't see guidance in the style guide about inline styles, but for me personally I'd prefer if this was in styles.css?

Reply:
Thank you!

Reply:
Happened to find the section that talks about it :) https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns